plugin-build/src/test/kotlin/io/sentry/android/gradle/SentryPropertiesFileProviderTest.kt

```diff
@@ -0,0 +1,239 @@
+package io.sentry.android.gradle
+
+import com.android.build.gradle.AppExtension
+import io.sentry.android.gradle.SentryPropertiesFileProvider.getPropertiesFilePath
+import org.gradle.api.Project
+import org.gradle.testfixtures.ProjectBuilder
+import org.junit.Test
+import java.io.File
+import kotlin.test.assertEquals
+
+class SentryPropertiesFileProviderTest {
+
+    private val sep = File.separator
+
+    @Test
+    fun `getPropertiesFilePath finds file inside debug folder`() {
+        val project = ProjectBuilder.builder().build()
+        project.plugins.apply("com.android.application")
+        val android = project.extensions.getByType(AppExtension::class.java).apply {
+            compileSdkVersion(30)
+        }
```

Comment:
looks like this block repeats across tests, maybe we create a method that returns a `project`?
it might require a param with a default value but allows customizing the `AppExtension` as some tests require only `compileSdkVersion(30)` and others with flavors and so on

Reply:
Yup added a:
```
    private fun createTestAndroidProject(
        parent: Project? = null,
        block: AppExtension.() -> Unit = {}
    ): Pair<Project, AppExtension> {
```

That will do the job 👌